src/components/game/GameTurnBegins.js

```diff
@@ -61,16 +63,20 @@ class GameTurnBegins extends React.Component {
   async componentDidMount() {
     try {
       const response = await api.get('game/mock');
-      this.setState({gameModel: response.data});
-      this.setState({playerIds: response.data.playerIds});
+      const gameModel = new GameModel(response.data);
```

Comment:
I would prefer to have gameModel (and probably also user) in the state instead of setting constant variables, like it was before. This way we already have them defined and can use them. And const has block-scope, meaning it can't be accessed from outside the function.

Reply:
So we should do it with "this.setState(..)" and not with the in advance defined models in an other js. file?

Reply:
What I meant was to define gameModel as null in the constructor (so we can check in the components if it already has the data), and only update it in componentDidMount. So this.setState({gameModel: new GameModel(response.data)}) instead of const gameModel = ...

Reply:
ah so we will use the defined gameModel.js but we will update the gameModel in the constructor....... Yes we can do this.

I will take a look tomorrow then I will merge the master in this branch so I can adjust the turnBegins to the conditional rendering..........